lib/Inspection/JsonPrintInspector.h

```diff
@@ -237,18 +237,15 @@ struct JsonPrintInspector
 
   void incrementIndentationLevel() {
     if (_format == JsonPrintFormat::kPretty) {
-      for (unsigned i = 0; i < IndentationPerLevel; ++i) {
-        _indentation.push_back(' ');
-      }
+      _indentation.append(IndentationPerLevel, ' ');
     }
   }
 
   void decrementIndentationLevel() {
     if (_format == JsonPrintFormat::kPretty) {
-      assert(_indentation.size() >= IndentationPerLevel);
-      for (unsigned i = 0; i < IndentationPerLevel; ++i) {
-        _indentation.pop_back();
-      }
+      assert(_indentation.size() >= IndentationPerLevel &&
```

Comment:
Is assert, not tri_assert is international?

Reply:
Yes, because this is used in some smaller test libraries which should not depend on AssertionLogger etc.